src/main/java/boss/Boss.java

```diff
@@ -0,0 +1,74 @@
+package boss;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.IOException;
+
+/**
+ * Welcome! This class represents the main class of the Boss Chatbot.
+ */
+public class Boss {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public enum Types {
+        TODO, DEADLINE, EVENT, NONE
+    }
+
+    /**
+     * Starts the Chatbot.
+     */
+    public void run() {
+        ui.welcome();
+        Scanner myObj = new Scanner(System.in);
+
+        Parser parser = new Parser(storage, tasks, ui);
+
+        String task = myObj.nextLine();
+        while (!task.equals("bye")) {
+            try {
+                parser.handleCommand(task);
+            } catch (FileNotFoundException e) {
+                ui.showLoadingError(e);
+            } catch (IOException e) {
+                ui.showLoadingError(e);
+            } catch (BossException e) {
+                ui.showLoadingError(e);
+            }
+            task = myObj.nextLine();
+        }
+        ui.bye();
+
+    }
+
+
+    /**
```

Comment:
Nice use of comments across all your code! It makes it very readable and easy to understand.